acai-ios/CollectionViewCells/MenuTabCollectionViewCell.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  MenuTabCollectionViewCell.swift
+//  acai-ios
+//
+//  Created by Drew Dunne on 3/28/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+extension UIImage {
```

Comment:
can we put this in an `UIImage+Extension.swift` file

Reply:
Sorry meant to delete this.